airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/JsonSchemaType.java

```diff
@@ -87,6 +90,7 @@ public class JsonSchemaType {
       JsonSchemaType.builder(JsonSchemaPrimitive.STRING)
           .withLegacyAirbyteTypeProperty("big_integer")
           .build();
+  public static final JsonSchemaType JSONB = JsonSchemaType.builder(JsonSchemaPrimitive.JSONB).build();
```

Comment:
was talking to Edward and I think we should add a legacy_airbyte_type = "json" here for debuggability

Reply:
> legacy_airbyte_type = "json"

added